packages/starlight/style/print.css

```diff
@@ -1,10 +1,5 @@
 @media print {
-	/**
-	This is increased in specificity so it works even when injected into pages before regular styles.
-	TODO: Just use `:root` once we move to @layer.
-	Same goes for the use of `!important` in this file.
-	*/
-	:root[data-theme]:is(:root) {
+	:root {
 		/* Colours — force light mode when printing */
 		--sl-color-white: hsl(224, 10%, 10%);
 		--sl-color-gray-1: hsl(224, 14%, 16%);
```

Comment:
Very cool seeing all these styles cleaned up! 🎉 

Do you think we should add these in a `starlight.print` layer? Or is it intentional that these aren’t layered because we want them to override user styles?

Reply:
So far, for print styles, this is the approach I've been using with Cascade Layers so it easily override styles but I'll admit I may not have enough experience/feedback with print styles and layers. Definitely could be an area where with time, we get more feedback and switch to a different approach if needed.